Documentation/developer-setup.md

```diff
@@ -0,0 +1,38 @@
+## **How to install Cesium Gem**
```

Comment:
I added an overview similar to Unreal wording over there. I also updated `Add Cesium Gem to an O3DE Project` to include the instructions for a completely new project and for the existing `cesium for o3de samples`. The reason for two different setups is because when adding a Gem to a project, the engine modifies the project setting to include the absolute path of the Gem. So it's impossible to seamlessly distribute the existing sample without modifying the project setting again to point to the existing Gem in a different machine. It should be fixed in the O3DE development branch, but not in the installer. Beside that, I added a new session for opening the project VS solution too.